benchto-service/src/main/webapp/index.html

```diff
@@ -34,7 +34,6 @@
     <script src="js/filters.js"></script>
     <script src="js/main.js"></script>
 
-    <link rel="stylesheet" href="http://google-code-prettify.googlecode.com/svn/trunk/src/prettify.css">
```

Comment:
Shouldn't that be replaced by this?
https://cdn.rawgit.com/google/code-prettify/master/src/prettify.css